src/include/sof/fw-ready-metadata.h

```diff
@@ -12,5 +12,6 @@
 
 extern const struct sof_ipc_cc_version cc_version;
 extern const struct sof_ipc_probe_support probe_support;
+extern const struct sof_ipc_user_abi_version user_abi_version;
```

Comment:
would this be part of the information reported after FW_READY or as part of the extended manifest you rmentioned?

Reply:
*Note: this patch originally appears in #2374. Duplicated here as a dependency to allow early comments on the uuid work (which is mentioned in the description). So any conversation regarding that one should be probably moved to #2374.*

This new piece is completely symmetrical to other existing data attached to the fw_ready atm. So I added this to fw_ready ipc payload as well until extended manifest is implemented and then should be moved like other static fw-ready payload pieces (1. consistency, 2. complete bundle in one place to be moved).